Makefile

```diff
@@ -11,6 +11,7 @@ endif
 REPO=planetscale
 NAME=pscale
 BUILD_PKG=github.com/planetscale/cli/cmd/pscale
+GOLANG_CROSS_VERSION  ?= v1.18.3
```

Comment:
I think this is the version of goreleaser-cross. It looks like they try to follow Go's versioning, but changing this also changes the goreleaser version. Previously we could use a specific version, I believe we can't do that anymore.

I say let's rename this to `GORELEASE_CROSS_VERSION` to indicate that this is the version of the docker image.

Reply:
@fatih Updated!